packages/redux/src/actions.ts

```diff
@@ -42,7 +41,7 @@ export const setWorkspaceStorage: PayloadActionCreator<WorkspaceStorage> = works
   payload: workspaceStorage,
 });
 
-export const setCommandHistoryStorage: PayloadActionCreator<CommandHistoryStorage> = commandHistoryStorage => ({
+export const setCommandHistoryStorage: PayloadActionCreator<unknown> = commandHistoryStorage => ({
```

Comment:
I still don't understand why redux is being changed in this PR. Nothing touches/modifies/uses this code in this PR. If it were broken elsewhere, shouldn't it be causing issues elsewhere?

Looking at this locally, the issue lies with chart. It should not import redux. Redux should only be consumed by our apps I think

This change also removes type safety from this action

Reply:
Make a separate ticket.

`@deephaven/redux` should not have dependencies on other packages is the direction we should go in. In this case, the redux actions registered should be exported by the package that registers them (e.g. `dashboard-core-plugins`, or `dashboard`)

Reply:
cool cool cool... so is this approved then or what?

Reply:
Ticket created: https://github.com/deephaven/web-client-ui/issues/730